ui/templates/welcome.html

```diff
@@ -6,9 +6,26 @@
 
 <p>
```

Comment:
As discussed yesterday, I thought we didn't want to allow the user to go to the listing page when there are no imported courses. Does this change that? 

My understanding of the comments above is that the "Welcome" page will contain a single link per repo, which brings you to the (possibly blank) listing page, and the listing page will now have the import link. Is that correct?